src/api-helper/command/generator/generatorCommand.ts

```diff
@@ -220,6 +226,10 @@ async function computeStartingPoint<P extends SeedDrivenGeneratorParameters>(par
     }
 }
 
+export function isProjectPromise(a: any): a is Promise<Project> {
```

Comment:
I wouldn't export this. The typing is fragile. It returns true on anything that looks like any promise.